content/docs/oracle/run-oracle/_index.md

```diff
@@ -34,63 +35,141 @@ Transaction costs for submitting price report data on-chain _are_ refunded but t
 
 1. Enter your working directory for the oracle server.
 
-2. Configure data source plugins. Navigate to the `config` sub-directory of your installation (default: `./build/bin/config`) and edit the `plugins-conf.yml` file to add plugin configuration.
+2. Create and edit your oracle server config file `plugins-conf.yml` to specify the `name` and `key` for each plugins you are using.
 
-    For how to do this see [Configure plugins](/oracle/run-oracle/#configure-plugins) on this page.
+   ```bash
+   touch plugins-conf.yml
+   ```
 
-3. (Optional) Add your own data source plugin(s). If you have developed your own FX plugins, (a) add sub-directory(ies) containing the plugin source code to the `plugins` sub-directory of your installation; (b) add config entry(ies) to the `plugin-conf.yml` file. 
+   {{< alert title="Info" >}}
+   A [sample `plugins-conf.yml` config file <i class='fas fa-external-link-alt'></i>](https://github.com/autonity/autonity-oracle/blob/master/config/plugins-conf.yml) can be downloaded from the Autonity Oracle Server GitHub.
+   {{< /alert >}}
+
+   Edit `plugins-conf.yml` to [configure plugins](/oracle/run-oracle/#configure-plugins) for data sources. See [Set up plugins config file](/oracle/run-oracle/#set-up-plugins-config-file) for how to do this.
+
+3. (Optional) Add your own data source plugin(s). If you have developed your own FX plugins, (a) add sub-directory(ies) containing the plugin source code to the `plugins` sub-directory of your installation; (b) add config entry(ies) to the `plugins-conf.yml` file. 
  
 4. Configure the oracle server. Specify the oracle server configuration; see [command line reference](/reference/cli/oracle/). Options can be set as system environment variables or directly in the terminal.
 
 5. Start oracle server:
 
+
     ``` bash
     autoracle \
-        -oracle_key_file="<KEYFILE>" \
+        -oracle_key_file="<ORACLE_KEYFILE>" \
         -oracle_key_password="<PWD>" \
-        -oracle_autonity_ws_url="<WS_ADDRESS>" 
+        -oracle_autonity_ws_url="<WS_ADDRESS>"
         -oracle_plugin_conf="<PLUGINS_CONFIG_FILE>/plugins-conf.yml" ;
```

Comment:
The naming for flags are updated as well by according to the feedback from the game. Please update those examples in the doc by refering to: 
```shell
Usage of Autonity Oracle Server:
Sub commands: 
  version: print the version of the oracle server.
Flags:
  -key.file="./UTC--2023-02-27T09-10-19.592765887Z--b749d3d83376276ab4ddef2d9300fb5ce70ebafe": Set oracle server key file
  -key.password="123": Set the password to decrypt oracle server key file
  -log.level=2: Set the logging level, available levels are:  0: NoLevel, 1: Trace, 2:Debug, 3: Info, 4: Warn, 5: Error
  -plugin.conf="./plugins-conf.yml": Set the plugins' configuration file
  -plugin.dir="./plugins": Set the directory of the data plugins.
  -symbols="AUD-USD,CAD-USD,EUR-USD,GBP-USD,JPY-USD,SEK-USD,ATN-USD,NTN-USD,NTN-ATN": Set the symbols string separated by comma
  -tip=1: Set the gas priority fee cap to issue the oracle data report transactions.
  -ws="ws://127.0.0.1:8546": Set the WS-RPC server listening interface and port of the connected Autonity Client node
```

Reply:
Done, resolved in commit 7b5a773d9d7c2590fad714687b3a1fcdea79e704